### ISSUE TITLE
Oracle 12c model generation not working

### ISSUE THREAD

**Issue by santhanamts** [Kononnable/typeorm-model-generator#83]
When generating model for oracle 12c, getting below error

```
typeorm-model-generator@0.2.17
[13:54:17] Starting creation of model classes.
(node:7608) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: Cannot read property 'replace' of undefined
(node:7608) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by santhanamts** [comment]
The issue was because of oracle output format. 
https://oracle.github.io/node-oracledb/doc/api.html#propdboutformat

the output format must be `oracledb.outFormat = oracledb.OBJECT;`

Appreciate if you can fix this issue.

**Comment by Kononnable** [comment]
Fixed in `typeorm-model-generator@0.2.18`
